lib/protobuf/src/server.proto

```diff
@@ -81,6 +81,8 @@ message CatalogRequest {
 
 message CatalogEntry {
   string repo_name = 1;
+  int32 limit = 2;
+  string last_tag = 3;
```

Comment:
I'm a bit surprised to see this in the return type. I think it should be in the request, but not the return?